pygpu/gpuarray.pyx

```diff
@@ -40,6 +40,15 @@ cdef bytes _s(s):
         return s
     raise TypeError("Expected a string")
 
+cdef size_t countis(l, object val):
```

Comment:
I think this will be faster, even on older numpy since it compares with `is`, not `==`.  It's also C code so it won't be that slow in any case.